### ISSUE TITLE
QNAP Webinterface Crashed

### ISSUE THREAD

**Issue by silencshadow** [Yannik/qnap-letsencrypt#14]
Hello,

since i started renew_certificate.sh my Server stopped working, after reinstall entware not started and the Webinterface not too .. it not completet the renew ... got some errors and now i cant do anything...

**Comment by silencshadow** [comment]
Started python SimpleHTTPServer with pid 10577
Traceback (most recent call last):
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/SimpleHTTPServer.py", line 230, in <module>
    test()
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/SimpleHTTPServer.py", line 226, in test
    BaseHTTPServer.test(HandlerClass, ServerClass)
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/BaseHTTPServer.py", line 595, in test
    httpd = ServerClass(server_address, HandlerClass)
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/SocketServer.py", line 420, in __init__
    self.server_bind()
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/BaseHTTPServer.py", line 108, in server_bind
    SocketServer.TCPServer.server_bind(self)
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/SocketServer.py", line 434, in server_bind
    self.socket.bind(self.server_address)
  File "/share/CE_CACHEDEV1_DATA/.qpkg/Python/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Parsing account key...
Parsing CSR...
Registering account...
Traceback (most recent call last):
  File "acme-tiny/acme_tiny.py", line 198, in <module>
    main(sys.argv[1:])
  File "acme-tiny/acme_tiny.py", line 194, in main
    signed_crt = get_crt(args.account_key, args.csr, args.acme_dir, log=LOGGER, CA=args.ca)
  File "acme-tiny/acme_tiny.py", line 92, in get_crt
    raise ValueError("Error registering: {0} {1}".format(code, result))
ValueError: Error registering: 400 {
  "type": "urn:acme:error:malformed",
  "detail": "Provided agreement URL [https://letsencrypt.org/documents/LE-SA-v1.0.1-July-27-2015.pdf] does not match current agreement URL [https://letsencrypt.org/documents/LE-SA-v1.1.1-August-1-2016.pdf]",
  "status": 400
}

**Comment by silencshadow** [comment]
Ok, webinterface fixed, Error still exist ... i cant get the renew.sh work ...

**Comment by Yannik** [comment]
you need to update acme-tiny i guess.

**Comment by silencshadow** [comment]
The problem is, I got already certificates but I want to get them for a new address. So I outlined the date check in the renew and starter it again... Now Python is not stopping the server and I don't know how to delete the wellknown.. It stops at account registration. How do I clean install everything again or update my old certs to the new address. Would be nice to have a .sh for "new domain address" and is there a way to use it without port 80? Don't want to open that port ... Regards.
